app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue

```diff
@@ -48,6 +48,18 @@
             @click="onClickAssignAuthor"
           />
         </label>
+        <woot-input
+          v-model.trim="sequence"
```

Comment:
It's not suggested to use `.trim` in v-model. Let's just trim it before sending it via the API